chat/public/css/chat.css

```diff
@@ -25,21 +25,29 @@ body.view-in-course .course-wrapper.chromeless {
 
 .course-wrapper.chromeless .chat-block {
     width: 100vw;
+    display: block;
+    overflow: auto;
 }
 
-.chat-block .subject {
+.subject {
```

Comment:
This rule is not specific enough - it could have an effect on other class="subject" elements in the system. You should scope it to `.chat-wrapper .subject { ... }`.

Reply:
Thanks for catching this, will commit the fix now.